getting-started/deployment_guide.md

```diff
@@ -284,4 +289,37 @@ kubectl exec -it $POD -- bash -c "chroot"
 ```
 Any binary other than `bash` and `nginx` would be permission denied.
 
+If you don't see Permission denied please refer [here](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#debug-kubearmor-installation-issue-in-dockerized-kubernetes-environment) to debug this issue
+
+</details>
+
+<details>
+<summary><h4>Debug KubeArmor installation issue</h4></summary>
```

Comment:
I do not think we should have this section in the getting-started guide. Lets discuss this in our regular huddle call.